test/test/parser/Cpu.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.parser;
+
+import java.io.File;
+
+public class Cpu {
+    private static volatile int value;
+
+    private static void method1() throws Exception {
+        long startTime = System.currentTimeMillis();
+        while (System.currentTimeMillis() - startTime < 10) {
+            for (String s : new File("/tmp").list()) {
```

Comment:
For a simple test, I'd stay away from accessing file system.
`System.getProperties().hashCode()` might be a good example of a basic yet cpu consuming call.